tools/donate-cpu-server.py

```diff
@@ -46,28 +51,32 @@ def handle_uncaught_exception(exc_type, exc_value, exc_traceback):
 
 
 def strDateTime():
-    d = datetime.date.strftime(datetime.datetime.now().date(), '%Y-%m-%d')
-    t = datetime.time.strftime(datetime.datetime.now().time(), '%H:%M')
-    return d + ' ' + t
+    return datetime.datetime.now().strftime('%Y-%m-%d %H:%M')
+
 
+def dateTimeFromStr(datestr):
+    return datetime.datetime.strptime(datestr, '%Y-%m-%d %H:%M')
 
 def overviewReport():
     html = '<html><head><title>daca@home</title></head><body>\n'
     html += '<h1>daca@home</h1>\n'
     html += '<a href="crash.html">Crash report</a><br>\n'
+    html += '<a href="stale.html">Stale report</a><br>\n'
     html += '<a href="diff.html">Diff report</a><br>\n'
     html += '<a href="head.html">HEAD report</a><br>\n'
     html += '<a href="latest.html">Latest results</a><br>\n'
     html += '<a href="time.html">Time report</a><br>\n'
     html += '<a href="check_library_function_report.html">checkLibraryFunction report</a><br>\n'
     html += '<a href="check_library_noreturn_report.html">checkLibraryNoReturn report</a><br>\n'
     html += '<a href="check_library_use_ignore_report.html">checkLibraryUseIgnore report</a><br>\n'
+    html += '<br>\n'
+    html += 'version ' + SERVER_VERSION + '\n'
     html += '</body></html>'
     return html
 
 
 def fmt(a, b, c, d, e):
-    column_width = [15, 10, 5, 6, 6, 8]
+    column_width = [40, 10, 5, 6, 6, 8]
```

Comment:
40 characters for the first column is very long.
But let's see how it looks on a smartphone, if it is too long we can reduce it to 30 or so.

Reply:
It was like this for the crash report - it has less columns though. If you look at the latest report with such long package names the date time will be overwritten - it's not an issue in that report but in others.

Reply:
It was intended by @danmar that for very long package names the date (not the time) is skipped since it is shown for many other packages and is normally the same. If there is a package with an extremely long name the rest of the columns are moved to the right so no important information is removed. Lets see how it works now.